apps/sso/views/register_new_user.jag

```diff
@@ -89,8 +89,39 @@ var themeModule = require('/modules/theme.js');
                                     </div>
 
                                 <%
+                                    function getSsoConfig(tenantId) {
+                                        var server = require('store').server,
+                                                registry = server.systemRegistry(tenantId);
+                                        return JSON.parse(registry.content(SSO_TENANT_CONFIG_PATH));
+                                    }
+
+                                    function getTenantId(referrerURI) {
+                                        var constants = require('rxt').constants;
+                                        var urlTenantId = constants.DEFAULT_TENANT;
+
+                                        if (!referrerURI) {
+                                            <!-- If directly copy registration URL to the browser or in similar kind of event TODO: Use  tenanted URL pattern for registration as well -->
+                                            return urlTenantId;
+                                        }
+
+                                        var host = referrerURI.split('/')[2];
+                                        referrerURI = referrerURI.split(host)[1];
+                                        var uriMater = new URIMatcher(referrerURI);
+                                        var elements = uriMater.match(constants.TENANT_URL_PATTERN) || uriMater.match(constants.DEFAULT_SUPER_TENANT_URL_PATTERN);
+                                        var domain = elements.domain;
+                                        var carbon = require('carbon');
+                                        urlTenantId = carbon.server.tenantId({domain: domain}); //TODO: change this logic once tenanted URLs are introduced to publisher as well
```

Comment:
Thanks @splinter , I have removed it in [this commit](https://github.com/wso2/carbon-store/pull/844/commits/5d5d71ad3588be9d70916fe117453be8d8f37bd9)